src/Sylius/Bundle/CoreBundle/Resources/config/app/config.yml

```diff
@@ -29,6 +29,12 @@ doctrine:
         entity_managers:
             default:
                 auto_mapping: true
+                mappings:
+                    gedmo_loggable:
+                        type: annotation
+                        prefix: Sylius\Bundle\ResourceBundle\Model
+                        dir: "%kernel.root_dir%/../src/Sylius/Bundle/ResourceBundle/Model"
```

Comment:
Indeed, it breaks every project using Sylius as a dependency, which is a problem (https://github.com/Sylius/Sylius/issues/7156). Yet it was merged ?